rc/filetype/just.kak

```diff
@@ -38,18 +38,19 @@ define-command -hidden just-indent-on-new-line %{
 # ‾‾‾‾‾‾‾‾‾‾‾‾
 
 add-highlighter shared/justfile regions
+
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/content/recipe regex '^@?([\w-]+)((?:\h[\w-]+(?:=(?:"[^\n]*?"))?)*):(?!=)(\h(?:[\w-]+|\([\w-]+\h[\w-]+\)))?' 1:function 2:value 3:type
+add-highlighter shared/justfile/content/globals regex ^([\w-]+)\h*:=\h*(["|'][^\n]*?["|'])((?:\h*\+\h*[\w-]+)*) 1:meta 2:value 3:meta
```

Comment:
The [grammar](https://github.com/casey/just/blob/master/GRAMMAR.md) calls them "assignments".